src/charm.py

```diff
@@ -86,11 +87,13 @@ def on_config_changed_handler(event, fw_adapter):
     pod_is_ready = False
 
     while not pod_is_ready:
-        logging.debug(f"Checking k8s pod readiness")
+        logging.debug("Checking k8s pod readiness")
         k8s_pod_status = k8s.get_pod_status(juju_model=juju_model,
                                             juju_app=juju_app,
                                             juju_unit=juju_unit)
+        logging.debug("Received k8s pod status: %s" % k8s_pod_status)
```

Comment:
We should probably standardize on a single string formatting method. There are currently three ways:

1. f-strings (`f"static{some_variable}"`)
2. format method (`"static{}".format(some_variable)`)
3. old style (`"static %s" % some_variable`)

It looks like the framework team is standardizing on #2 since it's the most modern AND is still compatible with Python 3.5.

Reply:
True. Let me fix that.